src/main/java/org/terasology/web/resources/systemStatus/SystemResource.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2017 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.web.resources.systemStatus;
+
+import org.terasology.web.resources.base.AbstractSimpleResource;
+import org.terasology.web.resources.base.ClientSecurityRequirements;
+import org.terasology.web.resources.base.ResourceAccessException;
+import org.terasology.web.resources.base.ResourceMethod;
+import org.terasology.web.resources.base.ResourcePath;
+
+import static org.terasology.web.resources.base.ResourceMethodFactory.createParameterlessMethod;
+
+public class SystemResource extends AbstractSimpleResource {
```

Comment:
I tested the whole server+frontend setup and it mostly works as expected, except one detail - the frontend shows the values received at the moment of the connection, and they are not updated. I noticed you added `system` as one of the resources the home tab is subscribed to, so the frontend should be ready to receive updates without asking for them; now we need to have the server actually send these updates.

I'd suggest you take a look (if you haven't already), for example, at `OnlinePlayersResource`. As you can see, when the value changes and you want to send the update to all the connected clients, you just have to call `notifyChangedForAllClients()` (which is a method of the `AbstractSimpleResource` class).

The only difference is that `OnlinePlayersResource` knows when there is an update to send: when it receives an event from the engine which says a player has joined or disconnected. On the other hand, `SystemResource` doesn't really know when to send updated stats, since they are changing all the time. I'd suggest to send them periodically, for example every 5 seconds (or maybe we can make it configurable).

To implement this you may use a thread which runs in the background and runs an infinite loop in which it sleeps for a certain amount of seconds, then calls `notifyChangedForAllClients()`. But there are probably better ways, for example there is a class called `ScheduledExecutorService` which if I understand correctly should be well-suited for this purpose. Googling for "java schedule periodic task" leads to various interesting results ([here is one](https://stackoverflow.com/questions/7814089/how-to-schedule-a-periodic-task-in-java)).

The whole resource handling system **should** be thread safe - I think calls to `onConnected` and `onDisconnected` in `OnlinePlayersResource` also come from a separate thread (not the same one the web server is running in) - however in case you run into problems I think we can merge the pull request at this point and fix the push notifications later.